app/AppKernel.php

```diff
@@ -25,6 +25,7 @@ public function registerBundles()
             new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle(),
             new Knp\Bundle\MenuBundle\KnpMenuBundle(),
             new Eo\AirbrakeBundle\EoAirbrakeBundle(),
+            new SimpleBus\SymfonyBridge\SimpleBusCommandBusBundle(),
```

Comment:
@carakas I only included the minimal requisites to be able to use the command bus. But maybe it is a good idea to initialise the doctrine bundle as well, because it is quite handy to use the rollback functionality.